style.qss

```diff
@@ -29,7 +29,7 @@ QWidget {
 #step_filters_frame QPushButton {
     font-size: 9px;
     margin: 5px;
-    background-color: transparent;
+    background-color: #515151;
```

Comment:
nitpick: is there a way we could avoid adding color constants to the QSS? Can we rely on the underlying style to get it? What is the normal bg col for `QPushButton`? Maybe worth adding a comment explaining why we need to set this to a dark gray.

Reply:
I changed it to `none` and it seems to behave (Maya, Nuke, Photoshop).